ksql-engine/src/test/java/io/confluent/ksql/EndToEndEngineTestUtil.java

```diff
@@ -833,11 +841,10 @@ private static KsqlEngine getKsqlEngine(final ServiceContext serviceContext) {
   static void shouldBuildAndExecuteQuery(final TestCase testCase) {
 
     final Map<String, Object> config = getConfigs(new HashMap<>());
-    final Properties streamsProperties = new Properties();
```

Comment:
Pointing this out for my own ease of understanding: this is the main functional change in this PR. Instead of creating `streamsProperties` from the configs, `streamsProperties` is instead created by the engine in `buildQuery` above and passed into this method.

Reply:
Yep that's correct.